### ISSUE TITLE
The singularity is an unstable terrible engine designed to make engineering look incompetent

### ISSUE THREAD

**Issue by AllStickyGithub** [ParadiseSS13/Paradise#14901]
**Issue Description**:
Singularity engines are unstable and border on being unusable. The main cause of this is a combination of the energy the PA gives, the range of the PA on the 0 setting and the movement of the singularity. Thanks to the completely random movement of the singularity it might go out of range of the PA and lose its energy reverting to stage 1. This will cause containment to fail. If you are lucky the engine will stay in the safe containment area. If you are unlucky it moves towards matter recharging it which results in SINGULARITY LOOSE ENGINEERING WHY ARE YOU SO BAD!

**What did you expect to happen**: 
I expected it to work like the singularity engine from months ago. Back then you could set it up and forget. Like the tesla engine. It still had considerable drawbacks though. Like it EMPing the area around it constantly making it impossible to work on by borgs and IPCs. It emitting radiation making work on containment much harder. The need of special equipment to look at it and stand near it without either being stunned or flying towards the containment field. The extra option for hijackers to turn it loose just by overcharging it and keeping it under control using their power beacon. It was an engine with a lot of drawbacks already making it hard enough to convince people it was a good option. This was before whatever the fuck is going on with it now was happening.

**What happened instead**:
Every shift you set it up is a roll of the dice. Maybe it will run fine or maybe it will just go loose.

**Why is this bad/What are the consequences**:
I hear you saying, "Why don't you just check up on the engine every so often you lazy engineer?". This is not an option. Mainly because it is impossible to know how much charge the singularity currently has. For all you know you it was hanging around the south side of containment before you can to check up on it. Leaving it with little charge left and the possibility for it to go loose the moment you back is turned. The only viable solution is for someone to watch the engine constantly and keep track of its movement. So, they can fiddle with PA settings to keep the dumb thing active. This is very exciting gameplay as you can probably imagine. I used to be able to argue about which engine we would call in. Now there is absolutely no reason to ever pick a singularity. 

**Steps to reproduce the problem**:
Setup a singularity engine.

**When did the problem start happening**:
There was a fix PR #13187 but I remember the engine working fine after it. There was what I would call a golden age for the singularity where it would run in a stable fashion with the PA set to 1. This allowed for some fun and creative engine designs but caused some issue because nobody bothered to edit the wiki. This was quite a few months ago and I cannot find the PR for it. I can find some wiki edits related to it in April and May of this year.

**Extra information**:

**Comment by Muhdi345** [comment]
Honestly, I don't think the PA not hitting it is the real cause of the Singuloth reverting to stage 1.

I've watched multiple times as the Singulo reverted to stage 2 from stage 3 on a power 2 PA. It's like the ticks that it loses power are doubled. And if Singulo rng was a factor for reverting to stage 1 I'd have it a lot more in my CE shifts in the late game.

Maybe I should ask an admin to look at it once I've determined the singulo will revert to stage 1 in one of my shifts...
